docs/spec.md

```diff
@@ -19,6 +19,7 @@ only a pointer file is written.
 * Values MUST NOT contain return or newline characters.
 * Pointer files MUST be stored in Git with their executable bit matching that
 of the replaced file.
+* Pointer files must be less than 1024 bytes in size.
```

Comment:
Thank you very much for adding this key bit of documentation!

The only thing I can think we might expand on here is to clarify that this maximum includes any Git LFS pointer extension lines as well, maybe just by saying something like:
```suggestion
* Pointer files must be less than 1024 bytes in size, including any pointer extension lines.
```

Reply:
I've squashed that in.